src/workerd/io/hibernation-manager.h

```diff
@@ -176,9 +182,27 @@ class HibernationManagerImpl final : public Worker::Actor::HibernationManager {
   uint16_t hibernationEventType;
   // Passed to HibernatableWebSocket custom event as the typeId.
 
-  kj::Maybe<HibernatableWebSocket&> webSocketForEventHandler;
-  // Allows the HibernatableWebSocket event handler that is currently running to access the
-  // HibernatableWebSocket that it needs to execute.
+  const uint16_t EVENT_QUEUE_CAPACITY = 1024;
```

Comment:
I don't know what a good value for this is, the previous assumption of 1 was too low, I feel like 1024 is pretty high but it I can imagine scenario's where we could go higher (you have 32k websockets that all message/close/whatever at once). It really depends on the latency between calling `customEvent()` and then actually running it.